app/locales/en/translations.js

```diff
@@ -295,7 +297,73 @@ export default {
     },
     "purpose_section": {
       "title": "Purpose(s)*"
+    },
+    "client_information": {
+      "title": "Client Information",
+      "id_type_held_by_client": "Type of ID held by client",
+      "verify": "We will use this ID to verify client on arrival.",
+      "data_confidentiality": "All client data is kept strictly confidential.",
+      "hk_id": "Hong Kong Identity Card",
+      "asrf_id": "Asylum Seeker Recognizance Form",
+      "client_first_name": "Client's First Name",
+      "client_family_name": "Client's Family Name",
+      "name_instruction": "As shown on identity document",
+      "name_warning": "You must enter your name here.",
+      "clients_phone_number": "Clients contact phone number",
+      "clients_phone_number_instruction": "We will use this number to confirm schedule and details.",
```

Comment:
This all will be `client_information. value2` so having `client` in value 2 is quite redundant.

Reply:
@Darshan-Chauhan , You mean it should be `client_information.number_instruction` instead of `client_information. clients_phone_number _instruction` ?